dts/bindings/mdio/zephyr,mdio-i2c.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright The Zephyr Project Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+description: Zephyr MDIO bitbang driver
```

Comment:
1) it's not a bitbang driver
2) "driver" is inappropriate term for DT binding description IMO
3) I don't think it's zephyr specific, maybe doesn't need prefix.

Reply:
maybe what would help here is do you have an actual hardware example that this runs on?  a binding could be made and associated to the actual hardware which this would run on, rather than being named after a common zephyr driver